File: techweek/practice/io_example.py
# PNU TECHWEEK Competition
# Practice Session Python I/O Example


# Get N from standard input
N = int(input())

# Declare list to contain vote results
votes = []

# get each vote result in 'votes' list with for loop
for i in range(N):
    x = input()
    votes.append(int(x))
    
# calculate answer

# TO-DO part
# Uses votes[0] as a temporary answer
answer = votes[0]

# print answer
print(answer)

File: techweek/practice/io_example.cpp
/*
 * PNU TECHWEEK Competition
 * Practice Session Python I/O Example
 */

#include <iostream>
using namespace std;

int main() {

    // Get N from standard input
    int N;
    cin >> N;

    // Declare array to contain vote results
    int votes[101];

    // get each vote result in 'votes' array with for loop
    for(int i=0; i<N; i++)
        cin >> votes[i];
        
    // calculate answer

    /*
    TO-DO part
    Uses votes[0] as a temporary answer
    */
    int answer = votes[0];

    // print answer
    cout << answer;

}